src/components/Modal/WalletNotice.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <UiModal :open="open" @close="$emit('close')">
+    <template v-slot:header>
+      <h3>{{ $t('walletNotice') }}</h3>
+    </template>
+    <div class="text-center my-2 p-4">
+      <h4>{{ $t('gnosisSafeWalletNotice') }}</h4>
+      <a
+        href="https://help.gnosis-safe.io/en/articles/4820197-how-to-participate-in-a-snapshot-poll"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <UiText :text="$t('learnMore')" />
```

Comment:
What the reason for UIText component here instead of just `{{ $t('learnMore') }}` ?

Reply:
if I remove `UiText`, There is a very small difference in the text displayed. i thought  to maintain it the same as other models.. (terms modal while voting), will remove if not required :)